docs/usage.rst

```diff
@@ -8,6 +8,14 @@ First, make sure you're using Python 3,
 you've :doc:`installed the bigchaindb_driver Python package <quickstart>`,
 and you've :doc:`connected to a BigchainDB node or cluster <connect>`.
 
+For the sake of these examples, we'll assume:
+
+.. ipython::
+
+    In [0]: from bigchaindb_driver import BigchainDB
+
+    In [0]: bdb = BigchainDB('http://bdb-server:9984/api/v1')
```

Comment:
Mmm, not sure how the IPython directives work, but I imagine their scope is limited to their current page. Maybe @sbellem knows, but likely, we'll need to put this bit *somewhere* on this page.